.github/workflows/dev-lead.yml

```
@@ -272,3 +274,7 @@ jobs:
             "repos/$RELAY_REPO/dispatches" \
             --input -
           echo "::notice::Relayed CI failure for PR $RELAY_PR_NUMBER — dispatched dev-lead-ci-failure"
+thod POST \
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)